test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -236,6 +238,12 @@ def convert_date_time(cls, dt=None, fmt=None):
         if dt is None:
             return None
 
+        stdoffset = timedelta(seconds=-time.timezone)
+        if time.daylight:
+            dstoffset = timedelta(seconds=-time.altzone)
+        else:
+            dstoffset = stdoffset
+        offsetdiff = dstoffset - stdoffset
```

Comment:
ok, you fixed it yourself by the time i posted my review. I think we can make this simpler by just setting offsetdiff to timedelta(seconds=-time.altzone) - timedelta(seconds=-time.timezone) in all cases. When daylight savings is off, timedelta(seconds=-time.altzone) seems to get set to timedelta(seconds=-time.timezone).

Reply:
@agrawalravi90 - timedelta(seconds=-time.altzone) - timedelta(seconds=-time.timezone) would not work as it will not give the number of seconds.
We should have -
offsetdiff = (int)(timedelta(seconds=-time.altzone) - timedelta(seconds=-time.timezone)).total_seconds()
the typecast (int) may not be needed. but we would need total_seconds()

Reply:
@agrawalravi90 : I think making it generic may not work on DST timezone. 
Also Python doc say "time.altzone
The offset of the local DST timezone, in seconds west of UTC, if one is defined. This is negative if the local DST timezone is east of UTC (as in Western Europe, including the UK). **Only use this if daylight is nonzero**."  Shall we keep the changes as it is?

Reply:
Ah, I didn't read that part, yes, please keep the changes as is then.

Reply:
I think generic will work. I tried the below code on my VM and one in SV, the results were as expected.
from datetime import timedelta, datetime
import time
offsetdiff = (int)((timedelta(seconds=-time.altzone) - timedelta(seconds=-time.timezone)).total_seconds())
print offsetdiff

However, we should follow what the documentation says.